contracts/AirbroCampaignFactory.sol

```diff
@@ -14,14 +14,31 @@ contract AirbroCampaignFactory is AirdropAdmin {
     address public immutable treasury;
     // protocol fee for claiming dropCampaign rewards
     uint256 public claimFee = 2_000_000_000_000_000; // 0.002 ETH
+    // protocol fee for creating dropCampaigns
+    uint256 public creatorFee = 0; // 0.000 ETH
     uint16 public claimPeriodInDays = 60;
+    bool public beta = true;
 
+    address public immutable betaAddress = 0x185310a0C79A9389e5552E338214EA86F0ef0f33;
```

Comment:
use constant for case like this, immutable is for when it is passed in the constructor